### DISCUSSION TITLE
How to copy sheets from icepakdesign to maxwelldesign?

### DISCUSSION THREAD

**Discussion by aknsj** [ansys/pyaedt#4364] [Q&A] [answered]
How to copy sheets from icepakdesign to maxwelldesign?

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @aknsj ,

You can use copy_solids_bodies_from method like in this example:
https://aedt.docs.pyansys.com/version/stable/examples/06-Multiphysics/Hfss_Icepak_Coupling.html#sphx-glr-examples-06-multiphysics-hfss-icepak-coupling-py

There is one extra argument in this method that allows you to export the sheets:
copy_solid_bodies_from(design, object_list=None, no_vacuum=True, no_pec=True, **include_sheets**=False)

If you only want to transfer the sheets, you can pass directly the sheet names in objec_list.

**Reply by aknsj** [reply]
Thank you for your answer.
